src/Val.cc

```diff
@@ -646,9 +646,8 @@ static ZeekJson BuildJSON(Val* val, bool only_loggable=false, RE_Matcher* re=new
 
 		case TYPE_OPAQUE:
 			{
-			j = ZeekJson::object();
 			auto* oval = val->AsOpaqueVal();
-			j["opaque_type"] = OpaqueMgr::mgr()->TypeID(oval);
+			j = {"opaque_type", OpaqueMgr::mgr()->TypeID(oval)};
```

Comment:
I expected a map-like initializer to look like `{ {key, value} }`, not `{key, value}`, so thought this change might actually be making an array now instead of an object ?  This example seems to confirm:

```
ZeekJson j;
j = { {"one", "two"} };
printf("%s\n", j.dump().data());
j = {"one", "two"} ;
printf("%s\n", j.dump().data());
```

Outputs:

```
{"one":"two"}
["one","two"]
```

Reply:
Yep, you're right. I misread the docs. I think there's also a missing unit test there since none of them caught that.